### PR TITLE
Create duplicate_bug.md

### PR DESCRIPTION
**Pull request by abbylindsay**
Create a duplicate bug template to import a bug from another repo into the current repo.

### CODE REVIEW

**Review by marcderhammer — APPROVED** [review]

**Review by andrewalc — APPROVED** [review]

**Review by j93lim — APPROVED** [review]

**Review by rcole1025 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/recurring_bug.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Recurring Bug
+about: Bug seen before/across multiple projects
+title: 'Recurring Bug'
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/recurring_bug.md:4]: I don't think this is necessary. We should either a) re-use the Bug template and reference the old/original bug or b) re-open the old/original bug template. My preference would be to do b) as we can have the history in one place. Github will show the issue re-opened and then the work can resume there.

**Review by barchard — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/recurring_bug.md (hunk 1 above)
Comment (abbylindsay) [reply .github/ISSUE_TEMPLATE/recurring_bug.md:4]: let's discuss in the projects meeting today. in our meeting last week there was pretty strong support for this idea.

File: .github/ISSUE_TEMPLATE/duplicate_bug.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: "Duplicate Bug!"
+description: "Bug is present within multiple projets"
+title: "[Duplicate Bug]: "
+labels: ["duplicate bug"]
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/duplicate_bug.yml:4]: I think you can label this as "duplicate", "bug"? instead of 1?

**Review by barchard — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/duplicate_bug.yml (hunk 2 above)
Comment (obermillerk) [reply .github/ISSUE_TEMPLATE/duplicate_bug.yml:4]: I would say just `duplicate` and remove the bug language altogether, just duplicate issue.

Comment (obermillerk) [reply .github/ISSUE_TEMPLATE/duplicate_bug.yml:4]: The tags from the original issue will be copied

Comment (abbylindsay) [reply .github/ISSUE_TEMPLATE/duplicate_bug.yml:4]: okay i will do that thanks @obermillerk

**Pull request merged**